WordPress/gutenberg
# Add optional capture group to url regex in wp-env

## What?

Resolve issue where urls with query parameters won't work in wp-env using `wp-env.json` to include plugins or themes.

## Why?

This can cause issues where adding plugins from third party developers can cause issues if they don't have their plugin added to the WordPress marketplace. My experience has been with Gravity Forms. Solves issue #43199 

## How?

Change the regex string that validates urls to include an optional capture group post `.zip` that will pass if the string has query parameters.

## Testing Instructions





1. Create `wp-env.json` file with plugins attribute with url values that match old pattern `[url]/zip-name.zip` and new pattern `[url]/zip-name.zip?authAttr={attrValue}`
2. Run `wp-env start`
3. Ensure that both plugin urls have been included into project and have been activated

## Screenshots or screencast
